spec/support/harvest_item_helpers.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# Helper methods for creating harvest items in tests
+module HarvestItemHelper
+  # Example methods for harvest item creation
+  module Example
+    # @return [Enumerator] sequence starting from 1
+    def sequence
+      @sequence ||= (1..).each
+    end
+
+    # Creates a harvest item with specified validation results
+    # @param fixable [Integer] Number of fixable validation results to create
+    # @param not_fixable [Integer] Number of not fixable validation results to create
+    # @param sub_directories [String, nil] Optional subdirectories to append to the path
+    # @param audio [Boolean] Whether to create an associated audio recording
+    # @return [HarvestItem] The created harvest item
+    def create_with_validations(fixable: 0, not_fixable: 0, sub_directories: nil, audio: false)
+      validations = []
+      fixable.times do
+        validations << BawWorkers::Jobs::Harvest::ValidationResult.new(
+          status: :fixable,
+          name: :wascally_wabbit,
+          message: nil
+        )
+      end
+      not_fixable.times do
+        validations << BawWorkers::Jobs::Harvest::ValidationResult.new(
+          status: :not_fixable,
+          name: :kiww_the_wabbit,
+          message: nil
+        )
+      end
+
+      info = BawWorkers::Jobs::Harvest::Info.new(
+        validations:
+      )
+      path = generate_recording_name(Time.zone.now, suffix: sequence.next.to_s)
```

Comment:
i'm wondering why this was even needed?

i'm guessing because the timestamp resolution was too coarse at the seconds level and that resulted in duplicates?

maybe just take the timestamp value, pull out the `.subsec.to_f` and use that as the suffix?